channels/au.m3u

```diff
@@ -63,9 +63,9 @@ https://d9quh89lh7dtw.cloudfront.net/public-output/index.m3u8
 https://austchannel-live.akamaized.net/hls/live/2002729/austchannel-news/master.m3u8
 #EXTINF:-1 tvg-id="BountyPlus.au" tvg-name="Bounty Plus" tvg-country="AU" tvg-language="English" tvg-logo="https://i.imgur.com/Q8mmLAB.png" group-title="",Bounty Plus (720p)
 https://bountyfilms-bounty-1-au.samsung.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="C31.au" tvg-name="C31" tvg-country="AU" tvg-language="English" tvg-logo="" group-title="",C31 (Melbourne) (404p) [Offline]
+#EXTINF:-1 tvg-id="C31Melbourne.au" tvg-name="C31Melbourne" tvg-country="AU" tvg-language="English" tvg-logo="" group-title="",C31 Melbourne (404p) [Offline]
 https://dcunilive47-lh.akamaihd.net/i/dclive_1@739220/master.m3u8
-#EXTINF:-1 tvg-id="Channel44.au" tvg-name="Channel 44" tvg-country="AU" tvg-language="English" tvg-logo="" group-title="",Channel 44
+#EXTINF:-1 tvg-id="Channel44.au" tvg-name="Channel 44" tvg-country="AU" tvg-language="English" tvg-logo="" group-title="",Channel 44 [Geo-blocked]
 https://iptv-all.lanesh4d0w.codes/australia/ch44
```

Comment:
They moved to Cloudfront : https://d1k6kax80wecy5.cloudfront.net/WFqZJc/index.m3u8